actions/terra-k8s-linter/policy/deployments/general_policies.rego

```diff
@@ -24,15 +24,8 @@ deny_required_deployment_labels[msg] {
   msg = sprintf("%s must include Kubernetes recommended labels", [name])
 }
 
-deny_replicas[msg] {
-  input.kind == "Deployment"
-  input.metadata.annotations["bio.terra.linter/replicas_exception"] != "enabled"
-  input.spec.replicas < min_required_replicas
-  msg := sprintf("Must have at least 3 replicas. %s has %d replicas.", [name, input.spec.replicas])
-}
-
 deny_revision_history[msg] {
   input.kind == "Deployment"
   input.spec.revisionHistoryLimit != revision_history_limit
-  msg := sprintf("%s should set revisionHistoryLimit to 0. Currently is: %b", [name, input.spec.revisionHistoryLimit])
+  msg = sprintf("%s should set revisionHistoryLimit to 0. Currently is: %d", [name, input.spec.revisionHistoryLimit])
```

Comment:
Smallest of nits: maybe we could use `revision_history_limit` here instead of hardcoding 0

Reply:
Not a nit-pick at all. e85d961